spec/requests/edit_work_spec.rb

```diff
@@ -147,27 +147,30 @@
           end
         end
 
-        context 'when editing globus' do
-          let(:work_version) { create(:work_version, :version_draft, :with_required_associations, work:) }
-          let(:upload_type) { 'globus' }
-          let(:fetch_globus_files) { 'true' }
-
-          it 'redirects to the work page and starts fetching globus' do
-            patch "/works/#{work.id}", params: { work: work_params, commit: 'Save as draft' }
-            expect(work.reload.head).to be_fetch_globus_version_draft
-            expect(response).to redirect_to(work)
+        context 'with a Globus upload type' do
+          before do
+            allow(GlobusClient).to receive(:get_filenames)
+              .and_return(["/uploads/#{user.sunetid}/work#{work.id}/version1/file1.txt"])
           end
```

Comment:
Changes to this test were added when the Globus client operation was added to the draft work form, so that the tests could keep passing.